simulator/config/config.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 	// ExternalImportEnabled indicates whether the simulator will import resources from a target cluster once
 	// when it's started.
 	ExternalImportEnabled bool
+	// ImportLabel is Labels to select which resources import from external cluster.
+	ImportLabel map[string]string
```

Comment:
can we use `metav1.LabelSelector`?
https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L1287